durexforth.asm

```diff
@@ -163,14 +160,17 @@ ONE
 !src "lowercase.asm"
 !src "disk.asm"
 
+    +BACKLINK "top", 3
+    +VALUE  TOP
+
 ; LATEST - points to the most recently defined dictionary word.
-    +BACKLINK
-    !byte 6
-    !text	"latest"
+; OBSOLETE!!! No user facing dict lists?

```

Comment:
The wordlist is useful, but `LATEST` is a lot less useful, as it only ever points to the top of the dictionary, and you can't navigate from there easily.  I think there need to be built-in words to manage the dictionary that will give no call for the user to get that address.

Reply:
I suppose it will remain useful - if nothing else, to figure out start of header space.

Reply:
We could save 2 bytes and a bunch of JSRs if LATEST were to be a value instead of a variable.  Given that the dictionary access idiom is changing, can LATEST change?

Reply:
I am not sure. It would be an API change, meaning major version bump. In practice though, a change like this will likely break some code due to the change in word structure, so a major version bump could be warranted anyway. What do you think?

Reply:
My reading of semantic versioning says yes, my gut says it's funny to release another major version so soon.

Reply:
I think it has to be yes :-)

Reply:
In that case, LATEST is now a value.